packages/core/src/api/tests/formatConversion/exportParseEquality/testExportParseEquality.test.ts

```diff
@@ -0,0 +1,78 @@
+import { describe, expect, it } from "vitest";
+
+import { BlockNoteEditor } from "../../../../editor/BlockNoteEditor.js";
+import { UnreachableCaseError } from "../../../../util/typescript.js";
+import { blockToNode } from "../../../nodeConversions/blockToNode.js";
+import { nodeToBlock } from "../../../nodeConversions/nodeToBlock.js";
+import { setupTestEditor } from "../../setupTestEditor.js";
+import {
+  TestBlockSchema,
+  TestInlineContentSchema,
+  TestStyleSchema,
+} from "../../testSchema.js";
+import {
+  addIdsToBlock,
+  partialBlocksToBlocksForTesting,
+} from "../formatConversionTestUtil.js";
+import {
+  ExportParseEqualityTestCase,
+  getExportParseEqualityTestCases,
+} from "./getExportParseEqualityTestCases.js";
+
+// Test for verifying that exporting blocks to another format, then importing
+// them back results in the same blocks as the original. Used broadly to ensure
+// that exporting and importing blocks does not result in any data loss.
+const testExportParseEqualityTest = async (
```

Comment:
should we not have this for markdown or html? or only for some of those cases?

Reply:
also, wdym with `Used broadly`?

Reply:
Added tests for HTML in #1605, though up until now HTML and Markdown wasn't really a concern since the export is lossy, so we didn't really expect the result after export + re-import to be the same as the original blocks.

Reply:
Also updated the comment